### ISSUE TITLE
Midea's "Inventor" brand de-humidifier additional controls missing (if supported)

### ISSUE THREAD

**Issue by mackonsti** [hillaliy/homebridge-midea-air#102]
Hello @hillaliy first of all, **warm** thanks for your great work. It was very hard to trace a device-controlling solution to you as I could not find the the appliance maker INVENTOR (www.inventor.ac) was part of MIDEA solutions and services...

So I am using latest Homebridge on an OpenMediaVault small sever (via Docker) I added your main plugin (just published, v1.6.10) via its Search interface and then added my Inventor iOS application credentials that were accepted, to my great surprise.

I selected (as part of testing) to use the "MideaAir" application in Settings and later, created a "child bridge" as Homebridge prompted me to do.

Reference: https://www.inventorairconditioner.com/dehumidifiers/eva-ii-pro-wi-fi-evp-wf
Model: Eva II Pro (EVP-WF20L) Dehumidifier with Wi-Fi

a) Is a **child bridge** necessary? If not, perhaps you could kindly consider to mention it in the first page README.md? Should I remove everything and re-install _without_ a child-bridge?

b) When I saved and restarted HomeBridge (as needed) my "Inventor" account was accepted and was logged in and I was surprised to see that my **appliance** was also detected, including my assigned appliance name! Well done:

```
[11/13/2022, 11:46:34 AM] Homebridge v1.5.1 (HAP v0.10.4) (Homebridge Server) is running on port 53307
[11/13/2022, 11:46:34 AM] [homebridge-midea-air] Launched child bridge with PID XXXX
[11/13/2022, 11:46:35 AM] Registering platform 'homebridge-midea-air.midea-air'
[11/13/2022, 11:46:35 AM] [homebridge-midea-air] Loaded homebridge-midea-air v1.6.10 child bridge successfully
[11/13/2022, 11:46:35 AM] Homebridge v1.5.1 (HAP v0.10.4) (homebridge-midea-air) is running on port 59342
[11/13/2022, 11:46:36 AM] [homebridge-midea-air] Created device: Eva II Pro, with ID: XXXXXXXXXXXXXX, and type: 161
```

(the ID I have obviously obfuscated, above)

On the devices list, the config icon shows the following and when long-pressing the tile, I get some options (that are not really related, i.e. the device is only a dehumidifier and does _not_ have a humidifier function (perhaps the orange button does nothing?)

At the moment, the only thing that can be done, is to **turn ON/OFF** and **see the reported humidity level** (with some delay, compared to the device display, next to me).

![Options](https://user-images.githubusercontent.com/1313613/201518474-f170585e-a83b-4b47-984b-23b0a2f01658.png)

Is there a way by providing some logs or tweaking the MideaAir config (that seems added successfully) to be able to control dehumidification modes, or fan speeds, or Ioniser ON/OFF etc.?

Or can I  provide specific logs to check with you whether the appliance does allow more to be controller, via your great plugin? Please kindly point me to the right direction, and I will try getting the log(s) necessary.

Many  thanks again.

P.S. What is the purpose of parameter `"supportedSwingMode": "both"` ? For A/C devices that "swing"? What are other options that I can manually set for this parameter as the appliance doesn't have any swings? Thanks!

P.S. How could I increase the responsiveness between the appliance-reported humidity and Homebridge UI?

![Config-Report](https://user-images.githubusercontent.com/1313613/201519204-a8f4263d-3f82-4c27-b4a4-4dbcd5a2b6ff.png)

**Comment by hillaliy** [comment]
Hi it's very difficult to fix all the problems. There are many models and Midea use an old and new protocol.
Also I don't have a dehumidifier to make tests. 
If You can give me access to your device I could try to make some adjustments

**Comment by mackonsti** [comment]
Hi @hillaliy thanks for replying.

A small request for text-label correction: Please on the report page above, do change the type label to `Humidifer-Dehumidifier` with a dash between the two types, so that people can understand a little better (like me 😄 )

How do I give you temporary access to my device? Via the Homebridge UI itself? Or provide the login credentials to the application?

Thanks.

**Comment by hillaliy** [comment]
About the label it's Homebridge label. Can't change it.
If you can send me your credentials.
My Discord user: Yossi#1563

**Comment by mackonsti** [comment]
Request to connect, sent. Many thanks

**Comment by hillaliy** [comment]
Fix on version 1.7.0
